### PR TITLE
unimatrix: add page

### PR DESCRIPTION
**Pull request by LordUbuntu**
- [X] The page(s) are in the correct platform directories: common, linux, osx, windows, sunos, android, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Unimatrix is cmatrix but with better support for unicode characters and various fonts. I thought it might be nice to add it here in case someone needs it. It's not an especially common program so if not that's just fine as well.

Thank you!

### CODE REVIEW

File: pages/common/unimatrix.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# unimatrix
+
+> A script to simulate the "Matrix" look with unicode characters.
+> See also: `cmatrix`
```
Comment (gutjuri) [inline pages/common/unimatrix.md:4]:
```suggestion
> See also: `cmatrix`.
```

File: pages/common/unimatrix.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+> More information: <https://github.com/will8211/unimatrix>.
+
+- Mimic default output of cmatrix (no unicode, works in TTY):
+
+`unimarix -n -s 96 -l o`
```
Comment (gutjuri) [inline pages/common/unimatrix.md:9]:
```suggestion
`unimatrix --no-bold --speed={{96}} --character-list={{o}}`
```

File: pages/common/unimatrix.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- No bold characters, slowly, with emojis, numbers, and a few symbols:
+
+`unimatrix -n -l ens -s 50`
```
Comment (gutjuri) [inline pages/common/unimatrix.md:13]:
```suggestion
`unimatrix --no-bold --character-list={{ens}} --speed={{50}}`
```

File: pages/common/unimatrix.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Change color or characters:
+
+`unimatrix -c {{COLOR}}`
+
+- Select chracter set(s) using a string over letter codes:
```
Comment (gutjuri) [inline pages/common/unimatrix.md:19]:
```suggestion
- Select character set(s) using a string over letter codes:
```

File: pages/common/unimatrix.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+`unimatrix -l {{CHARACTER_SET}}`
```
Comment (gutjuri) [inline pages/common/unimatrix.md:21]:
```suggestion
`unimatrix --character-list={{CHARACTER_SET}}`
```

File: pages/common/unimatrix.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+- Change scrolling speed:
+
+`unimatrix -s {{SPEED}}`
```
Comment (gutjuri) [inline pages/common/unimatrix.md:25]:
```suggestion
`unimatrix --speed={{SPEED}}`
```

File: pages/common/unimatrix.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# unimatrix
+
+> A script to simulate the "Matrix" look with unicode characters.
+> See also: `cmatrix`
+> More information: <https://github.com/will8211/unimatrix>.
+
+- Mimic default output of cmatrix (no unicode, works in TTY):
```
Comment (gutjuri) [inline pages/common/unimatrix.md:7]:
```suggestion
- Mimic the default output of cmatrix (no unicode, works in TTY):
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks for your contribution. In general, we prefer to display long options instead of short options.

File: pages/common/unimatrix.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# unimatrix
+
+> A script to simulate the "Matrix" look with unicode characters.
```
Comment (kbdharun) [inline pages/common/unimatrix.md:3]:
```suggestion
> A script to simulate the Matrix look with Unicode characters.
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution.

LGTM, after @gutjuri's suggestions.

**Comment by LordUbuntu** [comment]
Thanks for all the recco's, they've been merged.
And, Happy Spoopy Day!

File: pages/common/unimatrix.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+`unimatrix -c {{COLOR}}`
+
+- Select character set(s) using a string over letter codes:
+
+`unimatrix --character-list={{CHARACTER_SET}}`
+
+- Change scrolling speed:
+
+`unimatrix --speed={{SPEED}}`
```
Comment (acuteenvy) [inline pages/common/unimatrix.md:25]: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax

```suggestion
- Change the color of characters:

`unimatrix --color {{red|green|blue|white|...}}`

- Select character set(s) using letter codes (see `unimatrix --help` for available character sets):

`unimatrix --character-list {{character_sets}}`

- Change the scrolling speed:

`unimatrix --speed {{number}}`
```

File: pages/common/unimatrix.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
+# unimatrix
+
+> A script to simulate the Matrix look with Unicode characters.
```
Comment (acuteenvy) [inline pages/common/unimatrix.md:3]:
```suggestion
> Simulate the Matrix look with Unicode characters.
```

File: pages/common/unimatrix.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+> See also: `cmatrix`.
+> More information: <https://github.com/will8211/unimatrix>.
+
+- Mimic the default output of `cmatrix` (no unicode, works in TTY):
+
+`unimatrix --no-bold --speed={{96}} --character-list={{o}}`
```
Comment (acuteenvy) [inline pages/common/unimatrix.md:9]:
```suggestion
- Mimic the default output of `cmatrix` (no unicode, works in a TTY):

`unimatrix --no-bold --speed {{96}} --character-list {{o}}`
```

Comment (gutjuri) [reply pages/common/unimatrix.md:9]: Does unimatrix not require the equal signs? The documentation only gives examples with equal signs, so I'm not sure about this.

Comment (acuteenvy) [reply pages/common/unimatrix.md:9]: I tested it, `--opt arg` works exactly the same as `--opt=arg`.

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**